bedrock/firefox/urls.py

```diff
@@ -73,7 +73,7 @@
     # This dummy page definition makes it possible to link to /firefox/ (Bug 878068)
     url('^firefox/$', views.fx_home_redirect, name='firefox'),
 
-    url('^firefox/os/$', views.firefox_os_index, name='firefox.os.index'),
+    url('^firefox/os/$', views.fxos, name="fxos"),
```

Comment:
We typically name them like the `page()` helper would have. So for a template called "firefox/os/index.html" it would indeed be `firefox.os.index`. I like the more explicit and longer one personally. Gets confusing if we aren't consistent on such a large site.